linera-service/src/config.rs

```diff
@@ -445,7 +445,8 @@ Next Block Height:  {}"#,
 pub struct GenesisConfig {
     pub committee: CommitteeConfig,
     pub admin_id: ChainId,
-    pub chains: Vec<(ChainDescription, PublicKey, Amount, Timestamp)>,
+    pub timestamp: Timestamp,
+    pub chains: Vec<(u32, PublicKey, Amount)>,
```

Comment:
Maybe we can remove the `u32` in this case: those should be consecutive integers starting from 0.